subsys/spm/Kconfig

```diff
@@ -73,6 +74,15 @@ config SPM_SERVICE_REBOOT
 	help
 	  The Non-Secure Firmware is not allowed to reboot the chip. This
 	  service allows it to request a reboot from the SPM.
+
+config SPM_SERVICE_RNG
+	bool "Request random numbers"
+	default y
+	select NORDIC_SECURITY_BACKEND
+	select FLOAT
```

Comment:
Don't we need to enforce that mbedTLS must be enabled when SPM_SERVICE_RNG somehow?

Reply:
No. selecting NORDIC_SECURITY_BACKEND will not directly require configuring mbed TLS from Zephyr upstream. The library for nrf_cc310_mbedcrypto has the symbols required and the nrf_security module generates a corresponding mbed TLS configuration file for this use-case.

Reply:
aha, so NORDIC_SECURITY_BACKEND is enabling an implementation of the mbedTLS API?

Then this is good.

Reply:
Correct